Dshop/env-template

```diff
@@ -35,5 +35,6 @@ POSTGRES_PASSWORD=1234
 POSTGRES_HOST=db
 POSTGRES_PORT=5432
 
-STRIPE_PUBLISHABLE_KEY = stripe_publishable_key
-STRIPE_SECRET_KEY = stripe_secret_key
\ No newline at end of file
+STRIPE_PUBLISHABLE_KEY=stripe_publishable_key
```

Comment:
Suggestion: please update readme with info to check env-template and update .env in case of issues.